oceannavigator/frontend/src/components/DatasetSelector.jsx

```diff
@@ -539,6 +557,26 @@ function DatasetSelector({
       }}
     />
   ) : null;
+  let datasetSearchButton = null;
+  if (showSearchBtn) {
+    datasetSearchButton = (
+      <OverlayTrigger
+        key="draw-search-btn"
+        placement="top"
+        overlay={<Tooltip id={"draw-tooltip"}>{t("Search Datasets")}</Tooltip>}
+      >
+        <Button
+          variant="outline-primary"
+          size="sm"
+          onClick={toggleSearchDatasets}
+          title="Search Datasets"
+          className="ms-2"
+        >
```

Comment:
```suggestion
        <Button
          className="go-button"
          variant="primary"
          size="sm"
          onClick={toggleSearchDatasets}
          title="Search Datasets"
        >
```

Just to keep the styles consistent you can use the go buttons css class